### ISSUE TITLE
UILabel Font Size

### ISSUE THREAD

**Issue by abailac1** [luispadron/UICircularProgressRing#142]
Hi,

I have the last version, and didn't appear the option to set the inner label with different font size.
Anyway to do this programmatically?

![captura de pantalla 2018-09-26 a las 17 16 19](https://user-images.githubusercontent.com/43418544/46089940-e9c15800-c1af-11e8-9d2a-6ffcf2f9b452.png)

Regards

**Comment by abailac1** [comment]
ok, but in your GIF I can see that you manage the Font Size of the inner Uilabel...

Changin to Font Size = 36

![captura de pantalla 2018-09-26 a las 17 58 44](https://user-images.githubusercontent.com/43418544/46092730-d7e2b380-c1b5-11e8-9874-31c507b096d0.png)

Thanks.

**Comment by luispadron** [comment]
Right, this was when the API for this library was new and I used an integer for the font size. It has since moved to UIFont property, thus cannot be set in interface builder.
